command.go

```diff
@@ -31,6 +31,12 @@ import (
 // FParseErrWhitelist configures Flag parse errors to be ignored
 type FParseErrWhitelist flag.ParseErrorsWhitelist
 
+// Structure to manage groups for commands
+type Group struct {
+	Group string
+	Title string
+}
```

Comment:
I assume we create this struct to allow users to create a group once with a title, and then use the `Group.Group` field as a short id to be used multiple times?  Was that the idea behind this struct?

Reply:
I'm also a bit confused on this. Can't we just match on a string instead having to allocate a new "group" structure in memory?

Reply:
Yes, the idea was to use groups by an ID within the `Command` struct and have the title just set once. This also allows to rather easily change the title on just one place.

If you wish, I can remove the ID, but then we still need to save an array of Strings and call `AddGroup` for all groups to set the order of the groups.

Reply:
I think the separation between ID and Title may be good.  Let's first keep it, reviewe the other changes, and then see how we feel about it.